setup.py

```diff
@@ -13,5 +13,26 @@
     author_email='developers@savannahinformatics.com',
     url='',
     packages=find_packages(),
-    install_requires=[],
+    install_requires=[
+        "django==1.8.0",
```

Comment:
Use the "greater than but less than" style to constrain dependencies.

This style will keep us from getting security patches ( or require manual code changes ). 

Apply this principle - with discretion - on all the dependencies.